### PR TITLE
Reset to read file and updated input format

### PR DESCRIPTION
**Pull request by guntherstacey**

### CODE REVIEW

File: data-structure.py [hunk 1]
```diff
@@ -2,22 +2,9 @@ def read_file(text_file):
   if ".txt" in text_file:
     recipe_list = open(text_file)
     opened_recipe_list = (recipe_list.read())
-    return opened_recipe_list
+    print opened_recipe_list
   else:
     print ("Invalid Input: Not a .txt file")
 
+def data_structure(string_input):
```
Comment (sooca) [inline data-structure.py:9]: What is the need for an empty function here?

File: data-structure.py [hunk 2]
```diff
@@ -2,22 +2,9 @@ def read_file(text_file):
   if ".txt" in text_file:
     recipe_list = open(text_file)
     opened_recipe_list = (recipe_list.read())
-    return opened_recipe_list
+    print opened_recipe_list
```
Comment (sooca) [inline data-structure.py:5]: What is the reason for printing instead of returning file data here?

File: fridge-fill-sample-input.txt [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
-Warming Winter Curry: � $2 tins of coconut milk$, � $2 tins of chopped tomatoes$, � $18 small potatoes$ (600g), � $6 carrots$ (400g)\
-� $2 400g tins of haricot or cannellini beans$, � $1 bag of spinach$ (200g), � $3 teaspoons of turmeric$, -$3 teaspoons of ground cumin$\
-� $2 teaspoons of chilli flakes$, � $salt and pepper$#\
-Guacamole: $1 avocado$, $1/4 red onion$, $1/2 lime$, juiced, $1/4 tomato$, $handful of cilantro$, $salt$#\
-Salsa: $1 (28oz.) can whole plum tomatoes$, with the juice, $1 small white onion$, peeled and roughly chopped\
-$1-2 jalape�o peppers$, diced � Remove the seeds if you don�t like spicy salsa, $3 cloves garlic$, chopped, $2 Tablespoons lime juice$\
-$1-2 handfuls fresh cilantro$, roughly chopped, $1� teaspoons ground cumin$, $1 teaspoon salt$, $1 teaspoon granulated sugar, (optional)$#\
-Caramel Iced Coffee: $Handful of ice$, $1 cup almond milk$, $3 tablespoons caramel sauce$, $2 cups cold brew coffee$#"
+"Red Lentil Spinach Tikka Masala: 
+$2 tablespoons vegetable oil$ (I used coconut oil)
```
Comment (sooca) [inline fridge-fill-sample-input.txt]: Hey, looks much cleaner with new lines!

**Review by sooca — COMMENTED** [review]
Great first PR! Just a few comments here.

File: data-structure.py (hunk 1 above)
Comment (guntherstacey) [reply data-structure.py:9]: Hey!  This is why I put the empty function.

File: data-structure.py (hunk 2 above)
Comment (guntherstacey) [reply data-structure.py:5]: Because of this good reason I did it.

File: fridge-fill-sample-input.txt (hunk 3 above)
Comment (guntherstacey) [reply fridge-fill-sample-input.txt]: Thanks on this compliment!

**Review by sooca — APPROVED** [review]
Looks great! Great work!

**Pull request merged**